src/application/components/Explorer/GraphRefModal.tsx

```diff
@@ -29,207 +33,88 @@ export const GraphRefModal = ({
-                    css={{
-                      backgroundColor: "white",
-                      marginRight: "1rem",
-                      cursor: "pointer",
-                      padding: 8,
-                      width: 110,
-                      borderRadius: 4,
-                      fontWeight: 550,
-                      border: "1px solid #DEE2E7",
-                    }}
-                    onClick={onClose}
-                  >
-                    <span>Cancel</span>
-                  </button>
-                  <button
-                    css={{
-                      backgroundColor:
-                        !graphId || !graphVariant ? "#F4F6F8" : "#3f20ba",
-                      color: !graphId || !graphVariant ? "#777F8E" : "white",
-                      ...(!graphId || !graphVariant
-                        ? {}
-                        : { cursor: "pointer" }),
-                      padding: 8,
-                      width: 110,
-                      borderRadius: 4,
-                      fontWeight: 550,
-                      border: "1px solid #DEE2E7",
-                    }}
-                    disabled={!graphId || !graphVariant}
-                    onClick={() => {
-                      const newGraphRef = `${graphId}@${graphVariant}`;
-                      if (graphId && graphVariant && newGraphRef !== graphRef) {
-                        setNewGraphRefLoading(true);
-                        // if the graphRef has already been auth'd,
-                        // we can just pass it as a param, otherwise we must auth!
-                        if (hasGraphRefBeenAuthenticated(newGraphRef)) {
-                          setGraphRef(newGraphRef);
-                        } else {
-                          postMessageToEmbed({
-                            embeddedExplorerIFrame,
-                            message: {
-                              name: DEV_TOOLS_AUTHENTICATE_WITH_GRAPHREF,
-                              graphRef: newGraphRef,
-                            },
-                          });
-                        }
-                      }
-                    }}
-                  >
-                    Authorize
-                  </button>
-                </div>
-              </div>
-            </>
-          )}
-        </div>
-      </div>
-    </>
+            <div className="flex gap-2 justify-between mt-6">
+              <TextField
+                autoFocus
+                label="Graph id"
+                size="sm"
+                placeholder="acephei"
+                value={graphId}
+                onChange={(e) => setGraphId(e.target.value)}
+              />
+              <TextField
+                label="Graph variant"
+                size="sm"
+                name="value"
+                placeholder="current"
+                value={graphVariant}
+                onChange={(e) => setGraphVariant(e.target.value)}
+              />
+            </div>
```

Comment:
I've gone ahead and updated the layout so that the inputs stack when the screen gets narrow: [67f4d40](https://github.com/apollographql/apollo-client-devtools/pull/1180/commits/67f4d4025183c3ad8260d9ba5546079564e2150c)